### ISSUE TITLE
[PHP] Order of data type 

### ISSUE THREAD

**Issue by wing328** [swagger-api/swagger-codegen#3259]
##### Description

Currently, the data type in ObjectSerilaizer is not sorted in any order and it results in additional noise during code review
```
-        } elseif (in_array($class, array('integer', 'int', 'void', 'number', 'object', 'double', 'float', 'byte', 'DateTime', 'string', 'mixed', 'boolean', 'bool'))) {
+        } elseif (in_array($class, array('void', 'bool', 'string', 'double', 'byte', 'mixed', 'integer', 'float', 'int', 'DateTime', 'number', 'boolean', 'object'))) {
  ```
Ref: https://github.com/swagger-api/swagger-codegen/pull/3256/files#diff-d23a2c329903216ff7fe7a40f5c6b8d8R267

##### Swagger-codegen version

Last master (Jun 30, 2016)

##### Suggest a Fix

We want to keep the order fixed to avoid the additional noise.

(ideally the array should be replaced by hash/map (associated array in PHP) to improve the lookup time)

**Comment by arnested** [comment]
@wing328, did you actually fixed this in da45cb7ff515fd7cf10b142b7e23d58e2ab6f342 adding:
```java
        Collections.sort(sortedLanguageSpecificPrimitives);
```

**Comment by wing328** [comment]
@arnested yup
